.github/workflows/deploy.yml

```diff
@@ -105,8 +110,16 @@ jobs:
 
       - name: Update Cursor File
         uses: pleo-oss/pleo-spa-cicd/actions/cursor-deploy@v1.3.1
+        if: inputs.is_translation_files == false
+        with:
+          bucket_name: ${{ inputs.bucket_name }}
+
+      - name: Update Translation Cursor File
+        uses: pleo-io/pleo-spa-cicd/actions/translation-cursor-deploy@8f448cc666461aae07551240364b5c418df93863
```

Comment:
Version will be updated after merging https://github.com/pleo-oss/pleo-spa-cicd/pull/13